README.md

```diff
@@ -1,6 +1,6 @@
 # React Clock
 - Replace `<your_account>` with your Github username in the
- [DEMO LINK](https://<your_account>.github.io/react_clock/)
+ [DEMO LINK](https://sashalarin202.github.io/react_clock/)
```

Comment:
The last 2 buttons are redundant. https://prnt.sc/wp6arn
You need to have a button that changes the name of the clock, but you need to show in the console both previous and current name of the clock together, not just the current one.